embedded-redis/pom.xml

```diff
@@ -12,6 +12,20 @@
 
     <artifactId>embedded-redis</artifactId>
 
+    <properties>
+        <awaitility.version>3.1.6</awaitility.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.awaitility</groupId>
+                <artifactId>awaitility</artifactId>
+                <version>${awaitility.version}</version>
+            </dependency>
```

Comment:
can we have dependency management for this in parent pom?

Reply:
nobody is using it yet

Reply:
you actually don't need it, for the usecase you are using it -- there is wait strategy class

Reply:
removed